.travis.yml

```diff
@@ -27,4 +27,18 @@ install:
     - composer config extra.symfony.allow-contrib true
 
 script:
-    - composer req --ignore-platform-reqs $(echo $PACKAGES)
+    - |
+      EXIT_CODE=$(composer req --ignore-platform-reqs $(echo $PACKAGES));
+      echo "Exit code: $EXIT_CODE";
+      if [[ EXIT_CODE -eq 0 ]]; then
+        exit 0;
+      elif [[ EXIT_CODE -eq 1 ]]; then
+        echo "Could not install package";
+        exit 1;
+      elif [[ EXIT_CODE -eq 2 ]]; then
+        echo "Could not resolve dependencies. You can ignore this error if your package does not support Symfony $SKELETON_VERSION";
+        exit 1;
+      fi
+
+      exit $EXIT_CODE;
```

Comment:
I think the Travis output is more readable if we transform this into a function set in `before_script`, which is then used by the `script`:

```yaml
before_script:
  - |
      install_package() {
          composer req --ignore-platform-reqs $1
          local exit_code = $?
          echo "Exit code: $exit_code"
          if [[ exit_code -eq 0 ]]; then
            exit 0;
          elif [[ exit_code -eq 1 ]]; then
            echo "Could not install package";
            exit 1;
          elif [[ exit_code -eq 2 ]]; then
            echo "Could not resolve dependencies. You can ignore this error if your package does not support Symfony $SKELETON_VERSION";
            exit 1;
          fi
          exit $exit_code;
     }
     export -f install_package

script: install_package $(echo $PACKAGES)
```

Reply:
Yeah. You are correct. I'll keep updating. 

The script works now: https://travis-ci.org/github/symfony/recipes-contrib/jobs/701979394